BookingsApi/BookingsApi.DAL/Helper/CloneHearingToCommandMapper.cs

```diff
@@ -80,7 +80,7 @@ public static class CloneHearingToCommandMapper
 
         private static List<LinkedParticipantDto> GetLinkedParticipantDtos(Hearing hearing)
         {
-            var hearingParticipants = hearing.Participants.Where(x => x.LinkedParticipants.Any()).ToList();
```

Comment:
The method name doesn't represent what the implementation is doing. Are we returning linkedparticipants or interpreters?